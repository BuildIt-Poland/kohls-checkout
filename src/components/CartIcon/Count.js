import styled from 'styled-components';

import { colorWhite, fontWeightBold, fontSmall } from '../../styles/designTokens';

const Count = styled.div`
  position: absolute;
  color: ${colorWhite};
  font-weight: ${fontWeightBold};
  font-size: ${fontSmall};
  padding-bottom: 0.7rem;
`;

export default Count;
