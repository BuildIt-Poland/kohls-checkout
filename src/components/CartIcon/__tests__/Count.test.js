import React from 'react';
import { create } from 'react-test-renderer';

import Count from '../Count';

describe('COMPONENT - CartIcon Count', () => {
  it('renders correctly', () => {
    const component = create(<Count />);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
