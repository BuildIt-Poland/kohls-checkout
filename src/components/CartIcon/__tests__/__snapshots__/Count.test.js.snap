// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`COMPONENT - CartIcon Count renders correctly 1`] = `
.c0 {
  position: absolute;
  color: #ffffff;
  font-weight: 700;
  font-size: 1.2rem;
  padding-bottom: 0.7rem;
}

<div
  className="c0"
/>
`;
